scripts/release.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOT_DIR="${SCRIPT_DIR}/../"
+cd ${ROOT_DIR}
+
+print_usage() {
+  cat <<EOF
+Usage: $(basename $0) tag"
+Tag example: v1.2.3
+EOF
+}
+
+if [[ $# != 1 ]]
+then
+  print_usage
+  exit 1
+fi
+
+release_tag="$1"
+# without the starting 'v'
+release_version=$(echo "$release_tag" | cut -c2-)
+# 1 from 1.2.3
+major_version=$(echo "$release_version" | awk -F'.' '{print $1}')
```

Comment:
so how do we use these?

Reply:
This came from another script, it isn't needed. I was going to use them to do automatic version bumps but don't think that is a good idea anymore since this is based on a tag, which means the contents of the tag should already have the correct version in places like `lib/splunk/otel/version.rb`. I've removed them.